packages/veritone-widgets/src/shared/PropUpdater.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { func, object } from 'prop-types';
+
+export class PropUpdater extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.state.props = props.initialProps || {};
+  }
+
+  updateProps = newProps => {
```

Comment:
Is this getting used anywhere?

Reply:
That is the actual code :)

It's called from outside of component.